src/screens/Login.tsx

```diff
@@ -6,45 +6,45 @@ class Login extends React.Component {
   render() {
     return (
       <Container component="main" maxWidth="xs">
+        <Styles.MainDiv>
           <form noValidate>
-            <Styles.Label>Email</Styles.Label>
-            <Styles.Field
-              id='standard-secondary'
-              InputProps={{ disableUnderline: true }}
-              InputLabelProps={{ shrink: true }}
-              color="primary"
-              autoComplete="email"
-            />
-            <br></br>
-            <br></br>
-            <Styles.Label>Password</Styles.Label>
+            <Styles.FieldDiv>
+              <Styles.Label>Email</Styles.Label>
+              <Styles.Field
+                id='standard-secondary'
```

Comment:
make this id descriptive (this prop makes label and helperText accessible for screen readers)